psi4/src/psi4/libscf_solver/CMakeLists.txt

```diff
@@ -9,10 +9,11 @@ set(sources_list uhf.cc
                  stability.cc
 )
 psi4_add_module(lib scf_solver sources_list mints diis efp_solver fock)
-if(TARGET PCMSolver::PCMSolver)
+if(TARGET PCMSolver::pcm)
```

Comment:
Forks of forks can get a little crazy. I gave you write access to my fork, so do whichever you choose– push to my fork or use the web interface or whatever's most convenient. And thanks, I'm glad not to deal with HostWriter ever again.